Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -900,6 +900,6 @@
 #define ENABLE_SAMPLE_BUFFER_CONTENT_KEY_SESSION_SUPPORT 1
 #endif
 
-#if !defined(ENABLE_PREWARM_WEBPROCESS_ON_PROVISIONAL_LOAD) && PLATFORM(IOS)
+#if !defined(ENABLE_PREWARM_WEBPROCESS_ON_PROVISIONAL_LOAD) && (PLATFORM(IOS) || PLATFORM(MAC))
```

Comment:
Do we want to use `PLATFORM(COCOA)` so that we build the same code on all Apple platforms?

Reply:
That is a good point. I have not tested on watchOS yet, so perhaps we can land as-is, but we should unify this for all platforms. I can look into that in a follow up patch.

Thanks for reviewing!